how-we-work/epic-leads.md

```diff
@@ -0,0 +1,87 @@
+
+- Write a design document. If the Objective is substantial and will involve modification to the system architecture, you should write a design document and get it approved by the engineering team. The details on how to do that are in the [RFC document](../design-documents/0000-rfc-process/text.md).
+- Craft a project plan. If the Epic is substantial and there are many moving parts, you may want to create a short document on how you intend to stage the implementation of the child Epics (i.e. when and in what order). This is a way to help manage expectations so that PM might know when to review issues created for a child Epic, when something is planning on being pulled into a sprint, when a technical design doc will land, or even before the technical design phase, when you plan on completing some prototyping work.
+- Schedule a meeting with PM+UX to verify that the child issues that you have created for an Epic or child Epic actually implement the agreed upon product requirements (this should happen before the issues get placed into PokerBot for estimation).
+
+The Design Phase of an Epic is squishy and uncertain and so the path to the goals described above can be different each time. If you are struggling on how to find clarity or need any recommendations on next steps, reach out to your peers, ask PM/UX, or check in with your manager.
+
+## Implementation Phase
+
+Hooray! Now that you, PM, and UX have worked tirelessly to bring clarity to the project, now we can work together to bring it fruition. Note here that we work together as an engineering team to implement projects. Why? To reduce knowledge silos. Here are the scrum events you should be watching out for:
+
+### Refinement Meeting -- the Tuesday prior to the start of the sprint
+
+This is the most important refinement session for an Epic Lead.  This is the Refinement session that happens prior to pulling in the Epic into the sprint. This is when the tickets you have created will get estimated by the team.
+
+**Before** this refinement, there are the following expectations:
+
+- Tickets have been created that reflect the designed work.
+- Tickets have a complete definition of done. Any comments in the Github tickets have been consolidated/summarized in the top level comment. In short, tickets should be able to be estimated by the team without scrolling through a giant set of comments.
+- You have an idea of how to execute the work (i.e. where are the dependencies, what can be done in parallel, what will be demonstrated in the sprint demo). This is essentially the engineering plan to deliver the work. (PM will help model the plan in the Zenhub tickets prior to this meeting).
+- You have verified that the issues that you created represent the full scope of work with PM.
+- You have reached out to the Scrum Master the Monday before so that they can put up the issues on PokerBot for asynchronous estimation.
```

Comment:
Who's scrum master? We haven't really had a traditional scrum master.

Reply:
Ooh good catch! I've changed it to be "the manager" for now, but I suspect that this will change to "whoever is responsible for running Refinement."